### PR TITLE
Set up py3-only tools/quic and its venv

### PR DESCRIPTION
**Pull request by Hexcles**
This does not include the actual server, but only sets up the skeleton:
* Introduce a "py3only" to commands.json
* Create the skeleton of tools/quic and set up its venv
* `git subtree add --prefix tools/third_party/aioquic https://github.com/aiortc/aioquic.git 0.8.7 --squash` (note that this is checked in but not used at the moment)

The plan is to have `wpt serve` call `wpt servequic` (behind a flag).

For more details, see
https://github.com/web-platform-tests/rfcs/blob/master/rfcs/quic.md

Tracking issue: #19114

### CODE REVIEW

**Comment by Hexcles** [comment]
@jgraham you probably want to skip the two commits generated by `git subtree`.

File: tools/wpt/wpt.py [hunk 1]
```diff
@@ -95,7 +95,8 @@ def create_complete_parser():
     for command in commands:
         props = commands[command]
 
-        if props["virtualenv"]:
+        if props["virtualenv"] and (
+            (not props["py3only"]) or sys.version_info.major == 3):
```
Comment (Hexcles) [inline tools/wpt/wpt.py:99]: Otherwise, `pip install aioquic` will cause a fatal error in Python 2.

Comment (Hexcles) [reply tools/wpt/wpt.py:99]: Also, @jgraham I think this is the indentation style you prefer, but `flake8` is sad:
https://community-tc.services.mozilla.com/tasks/I9HcCed5SSqARezBCYTroQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FI9HcCed5SSqARezBCYTroQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L356

Comment (jgraham) [reply tools/wpt/wpt.py:99]: My preferred style would be:

```py
        if (props["virtualenv"] and
            (not props["py3only"] or sys.version_info.major == 3)):
```

File: wpt [hunk 2]
```diff
@@ -4,8 +4,10 @@ if __name__ == "__main__":
     from tools.wpt import wpt
     from sys import version_info, argv, exit
     args, extra = wpt.parse_args(argv[1:])
+    commands = wpt.load_commands()
+    py3only = commands[args.command]["py3only"]
 
-    if args.py3 and version_info.major < 3:
+    if (args.py3 or py3only) and version_info.major < 3:
         from subprocess import call
         exit(call(['python3', argv[0]] + [args.command] + extra))
```
Comment (jgraham) [inline wpt:12]: Might be good to improve the error handling here if `python3` isn't on the system, since there was no explicit opt-in.

**Review by jgraham — CHANGES_REQUESTED** [review]

**Comment by Hexcles** [comment]
@jgraham I've addressed your comments; PTAL again, thanks!

**Review by jgraham — APPROVED** [review]

**Pull request merged**